src/Auth0.ManagementApi/ManagementApiClient.cs

```diff
@@ -13,6 +13,7 @@ namespace Auth0.ManagementApi
     /// </summary>
     public class ManagementApiClient : IDisposable
     {
+        protected readonly IManagementConnection connection;
```

Comment:
I needed to add this as protected in order to be able to create a Test specific class that extends the ManagementApiClient to retry on RateLimit Exceptions.